src/taoensso/sente.cljx

```diff
@@ -81,6 +81,16 @@
   (:require-macros
    [cljs.core.async.macros :as asyncm :refer (go go-loop)]))
 
+#+cljs
+(def is-node? (nil? enc/js-?win))
```

Comment:
Does `*target*` come from the `:target` compiler option? If it does, that seems like the way to go. Otherwise, I think checking for `js/window` is the standard way to detect nodejs, but I can add an extra check for the `process` global var if you want.